common/static/common/js/discussion/views/discussion_thread_view.js

```diff
@@ -91,7 +91,6 @@
                     id = self.model.get('id');
                     if (collection.get(id)) {
                         self.model = collection.get(id);
```

Comment:
@bjacobel Just for the curiosity, What is this event handler, and why we need that?  
```
this.model.collection.on('reset', function(collection) {
    ...
    ...
}
```